src/parallel/GeneralDomainDecomposition.h

```diff
@@ -86,6 +90,11 @@ class GeneralDomainDecomposition : public DomainDecompMPIBase {
 	}
 
 private:
+	/**
+	 * Method that initializes the ALLLoadBalancer
+	 */
+	void initializeALL();
```

Comment:
Not sure about the software pattern here. If we had more Loadbalancer we probably would want a factory or facade here but for the moment this is fine.

Reply:
true